src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -16,6 +16,12 @@ public Mutect2FilteringEngine(final M2ArgumentCollection MTAC) {
         this.MTAC = MTAC;
     }
 
+    public void applyTriallelicFilter(final VariantContext vc, final Collection<String> filters) {
+        if (vc.getNAlleles() > 2) {
```

Comment:
Addressed in a later commit -- once this filter is reached only alleles passing TLOD and NLOD are in the vcf.  (I don't think of TLOD and NLOD as filtering, but rather as part of somatic genotyping).